src/components/ContactUs/Map.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { generate } from 'shortid'
+import GoogleMapReact from 'google-map-react'
+import styled from 'styled-components'
+import RatioContainer from '../Common/RatioContainer'
+import MapMarker from './Marker'
+import mapStyles from './mapStyles'
+import getLatLngMidpoint from '../../utils/getLatLngMidpoint'
+
+const StyledRatioContainer = styled(RatioContainer)`
+  > * {
+    position: absolute !important;
```

Comment:
👀

Reply:
ikr, to be fair its for the google maps element in the DOM which has inline styling applied to it, I could put an inner and outer wrapper on it but seems a bit of overkill...